README.md

```diff
@@ -238,6 +238,7 @@ class Missions
 * Tweaked: Sector defenders waypoint creation.
 * Tweaked: Daytime parameter values.
 * Tweaked: Shorter nights now multiplies the selected daytime time multiplier by 4 between 20:00 and 04:00. (3x between 21:00 and 03:00 before)
+* Tweaked: Supply crate position heights on M977 vehicles.
```

Comment:
This is not the changelog for the actual version.
As you want to contribute to the upcoming 0.96.6 version, you should've created a branch from that one as base :)
But don't worry and no need to hurry, take your time for this. Everything is fine and it's also your first PR :)

Reply:
You should rebase your branch on 0.96.6

Reply:
Yeah, I realized this after I commited the change. I don't think I need to revert it because I'll delete my cloned repo afterwards.
As I understand, I changed it in the Readme.md on this branch, too. At least GitHub says 2 files changed on this pr.

Reply:
Yeah, but in an old state and therefore under the wrong version in the changelog.
Have a look at the file from the 0.96.6 branch:
https://github.com/KillahPotatoes/KP-Liberation/tree/v0.96.6#0966-tbd